planet/src/main/webapp/js/authentication.js

```diff
@@ -48,13 +64,20 @@ function signUp() {
     const email = document.getElementById('email').value;
     const displayName = document.getElementById('first_name').value 
     + ' ' + document.getElementById('last_name').value;
+    const phoneNumber = document.getElementById('phone').value;
     resetForm('input-sign-up');
     if (password !== passwordConfirmation) {
         return;
     }
     firebase.auth().createUserWithEmailAndPassword(email, password)
     .then(function() {
-        return firebase.auth().currentUser.updateProfile({
+        const user = firebase.auth().currentUser;
+        database.ref('users/' + user.uid).set({
+            name: displayName,
+            email: email,
+            phoneNumber: phoneNumber,
```

Comment:
based on the documentation, it looks like this data is shared in realtime with all users, so this effectively shares the email and phone number of each user on the site with each other user.

Am I interpreting that correctly? If so, that won't satisfy our privacy requirements. We'll need to look for an alternate design.

Reply:
This is just an example of how to write data in the database. It will be removed later!